server/api/routes/subscriptions.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const { createSubscription } = require('../actions/subscriptions');
+
+// Create a new router object, where we will have CRUD routes for our subscriptions table 
+// under the route /users/:userId/subscriptions/
+const router = express.Router();
+
+// Read all user's subscriptions
+// Route: /users/:userId/subscriptions
+router.get(async (req, res) => {
+  try {
+    const data = await getSubscriptionsByUser();
+    res.status(200).json(data);
+  } catch(error) {
+    res.status(400).json(error);
+  }
+});
+
+// READ a single user subscription
+// Route: /users/:userId/subscriptions/:subscriptionId
+router.get(async (req, res) => {
```

Comment:
don't you need the url in this one?

```suggestion
router.get('/:subscriptionId', async (req, res) => {
```